circleci/dapple-deploy

```diff
@@ -26,6 +26,7 @@ DAPPLE_URL=$1
 DAPPLE_USER=$2
 DAPPLE_PASS=$3
 APP_NAME=$4
+# TODO: as part of INFRANG-4341 we would like to get rid of these 2 values. In the interim they will be ignored if the new launch config format is used.
```

Comment:
Just to confirm - the only change you intend for this PR is this comment?

Reply:
Yep. There'll be a followup PR after rolling out deploy config to get rid of the values.

But for now luckily the script should work as-is (and we can just do the ignoring in the circle-ci-integrations side)